src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLag;
+
+    public RightTriangle(String color, String name, int firstLag) {
+        super(color, name);
+        this.firstLag = firstLag;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (Math.pow(firstLag, 2) * (Math.sqrt(3)) / 4);
```

Comment:
The formula used in `calculateArea` is for an equilateral triangle, not a right triangle. For a right triangle, the area should be calculated as `(base * height) / 2`. Ensure that the correct dimensions are used for a right triangle.